app/pages/lab/visibility.cjsx

```diff
@@ -62,15 +99,41 @@ module.exports = React.createClass
       <div style={looksDisabled if @props.project.private or not @props.project.live}>
         <hr />
 
-        <p>
-          <button type="button" className="standard-button" disabled={@props.project.private or not @props.project.live or @state.setting.beta_requested or @props.project.beta_requested or @props.project.beta_approved} onClick={@set.bind this, 'beta_requested', true}>Apply for review</button>{' '}
-          {if @props.project.private
-            <span>Only <strong>public projects</strong> can apply for review.</span>
-          else if not @props.project.live
-            <span>Only <strong>live projects</strong> can apply for review.</span>
-          else if @props.project.beta_requested
-            <span>Review status has been applied for. <button type="button" disabled={@state.setting.beta_requested} onClick={@set.bind this, 'beta_requested', false}>Cancel application</button></span>}
-        </p>
+        <div>
+          {if @isReviewable()
+            <fieldset className="lab-visibility__policy-reviews">
+              <label>
+                <input type="checkbox" ref="labPolicyReviewed" onChange={@toggleCheckbox.bind this, 'labPolicyReviewed'} />
+                I have reviewed the <a href="/lab-policies" target="_blank">policies</a>
+              </label>
+
+              <label>
+                <input type="checkbox" ref="bestPracticesReviewed" onChange={@toggleCheckbox.bind this, 'bestPracticesReviewed'} />
+                I have reviewed the <a href="/lab-best-practices" target="_blank">best practices</a>
+              </label>
+
+              <label>
+                <input type="checkbox" ref="feedbackReviewed" onChange={@toggleCheckbox.bind this, 'feedbackReviewed'} />
+                I have reviewed the <a onClick={@showFeedbackForm}>review feedback form</a>
```

Comment:
Sure.  Any preference for styling?  `.minor-button` maybe looks a bit big?

<img width="388" alt="screen shot 2016-07-01 at 10 42 53 am" src="https://cloud.githubusercontent.com/assets/122990/16526807/a595d0e0-3f78-11e6-9bb5-6ad1c9a6bc4e.png">